mod/aalborg_theme/views/default/css/elements/misc.php

```diff
@@ -62,3 +62,22 @@
 	display: none;
 	padding: 0;
 }
+
+/* ***************************************
+	GLOBAL AJAX SPINNER
+*************************************** */
+
+.elgg-wait-cursor,
+.elgg-wait-cursor * {
+	cursor:wait !important
```

Comment:
Might be best to check the other properties also and remove anything that's not needed by Elgg.